tools/gn

```diff
@@ -1063,7 +1063,7 @@ def parse_args(args):
 
   parser.add_argument(
       '--stripped',
-      default=True,
+      default=False,
```

Comment:
Can we scope this only to Android?

Right now this is applying to all builds on all platforms.